### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by mkholjuraev**
Creates a PR template. Feel free to suggest improvements.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# After the change
+
+
+# Checklist:
+
+- [ ] The commit message has the Jira ticket linked
+- [ ] PR has a short description
+- [ ] Screenshots before and after the change are added
+- [ ] Tests for the changes have been added
+- [ ] Read.me is updated if necessary
```
Comment (bastilian) [inline .github/pull_request_template.md:24]: Maybe as an optional point in the checklist/testing requirements, we could add eventual dependencies. For example when something needs a PR from frontend-components or it needs to be run with another app together.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Description
+
+Associated Jira ticket: # (issue)
```
Comment (gkarat) [inline .github/pull_request_template.md:3]: Can we agree on having a full link? Going to a board and copy-pasting the id is slower that one click 😅

File: .github/pull_request_template.md (hunk 1 above)
Comment (gkarat) [inline .github/pull_request_template.md:24]:
```suggestion
- [ ] README.md is updated if necessary
```

**Review by gkarat — APPROVED** [review]
LGTM

File: .github/pull_request_template.md (hunk 2 above)
Comment (bastilian) [reply .github/pull_request_template.md:3]: Though do we really need it in the PR? If the issue ID has the right format and is in the commit message, GitHub should turn it into a link on the commit view. That is at least how I always go to Jira issues from PRs.

File: .github/pull_request_template.md (hunk 1 above)
Comment (mkholjuraev) [reply .github/pull_request_template.md:24]: @bastilian it makes sense. What would be the best wording for this?

File: .github/pull_request_template.md (hunk 2 above)
Comment (gkarat) [reply .github/pull_request_template.md:3]: yes, in PR it is not strictly needed (I also look at commits, not PRs), let's mention only URLs in commits

**Pull request merged**